### PR TITLE
Tofu html editor bug

### PR DESCRIPTION
**Pull request by eugeneoh28**
Bug to address issue of ng-model not binding to the code view.

### CODE REVIEW

File: source.js [hunk 1]
```diff
@@ -113,7 +118,8 @@
 
 				var code = this.source.$textarea.val();
 
-				code = this.paragraphize.load(code);
+				// Disable to workaround bug in paragraphize, which will keep adding <p> wrapper for some top level element such as <a>
+				// code = this.paragraphize.load(code);
```
Comment (patrick-minton) [inline source.js:123]: Are you sure there are no bad side effects from disabling this? I think we need to identify the use case for which this was first enabled, and verify that this isn't a problem.

**Review by patrick-minton — CHANGES_REQUESTED** [review]

File: source.js [hunk 2]
```diff
@@ -22,6 +22,11 @@
 				};
 
 				this.source.$textarea = $('<textarea />');
+				// This line here is a fix so that we can use ng-model binding for the code view
+				// for the redactor class. It basically grabs the original textarea field and binds
+				// it to the text area of the code view in the redactor so that the ng-model bind will
+				// be bound to the code view as well as the rich content editor view.
+				this.source.$textarea[0] = this.core.box()[0].childNodes[3];
```
Comment (leorojas86) [inline source.js]: is there a way to find the node by name/id/tag or something else?, finding element this way (by indexing arrays) is kind of error prone.

Comment (patrick-minton) [reply source.js]: this is a good point. It's a good practice to add a class name for it that's only for javascript, e.g. `js-my-class`

File: source.js (hunk 1 above)
Comment (eugeneoh28) [reply source.js:123]: Sorry about that, I actually didn't write this part of the code it was brought in from a different commit that was from a while ago but for some reason became apart of my branch

**Pull request closed without merging**